src/lib.rs

```diff
@@ -65,3 +64,5 @@ pub type Error = failure::Error;
 pub type Result<T> = std::result::Result<T, Error>;
 
 pub use crate::model::Model;
+
+pub use ::collection;
```

Comment:
Do we need to expose that as `pub`?

Reply:
probably no. I left like before. 
We could make it private once the new crate `collection` or whatever is published

Reply:
Oh, right, I forgot collection was exposed so better keep that way for now.